### DISCUSSION TITLE
GitHub Enterprise, base permissions, and GitHub apps access

### DISCUSSION THREAD

**Discussion by davidwinter** [community/community#38470] [Enterprise]
I was hoping to change our GitHub Enterprise organisation base permissions to "No permission" so that organisation members can't see any private repos by default, not unless they or the team they are assigned to have been explicitly added to a private repo.

If I make this change, the visibility for organisation members works as expected, however, connected GitHub apps like Slack or render.com are then are unable to connect to repositories that they have explicitly been granted permissions to access - completely breaking those integrations. So Slack notifications stop, and render.com deployments fail as they can't access the code anymore.

Is this intended? Is there a workaround?

**Comment by ossanna16** [comment]
Hi there @davidwinter and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
